vector/src/main/res/values-fr/strings_tchap.xml

```diff
@@ -76,30 +75,21 @@
     <string name="tchap_expired_account_on_new_sent_email_msg">Un nouvel e\u2011mail vous a été envoyé pour renouveler la validité de votre compte. Une fois que vous aurez suivi le lien qu’il contient, cliquez ci-dessous.</string>
 
     <!-- Login/Register Tchap -->
-    <string name="tchap_register_title">Inscription ${app_name}</string>
-    <string name="tchap_connection_title">Connexion ${app_name}</string>
     <string name="tchap_has_account_no">Je n’ai pas\nde compte</string>
     <string name="tchap_has_account_yes">J’ai un compte</string>
     <string name="tchap_connection_email">Adresse email</string>
     <string name="tchap_connection_email_help">Utilisez votre adresse professionnelle</string>
-    <string name="tchap_connection_password">Mot de passe ${app_name}</string>
     <string name="tchap_connection_password_help">Votre mot de passe doit contenir au moins 8 caractères, avec au moins un caractère de chaque type : majuscule, minuscule, chiffre, caractère spécial.</string>
     <string name="tchap_connection_password_confirm">Confirmer le mot de passe</string>
-    <string name="tchap_register_warning_for_external_title">Information concernant votre inscription</string>
-    <string name="tchap_register_warning_for_external">Le domaine de votre adresse e\u2011mail n’est pas déclaré dans ${app_name}. Si vous avez reçu une invitation, vous allez pouvoir créer un compte ${app_name} « invité », permettant uniquement de participer aux échanges privés auxquels vous êtes convié.</string>
-    <string name="tchap_register_warning_for_external_proceed">Poursuivre</string>
     <string name="tchap_register_wait_for_email_title">Un e\u2011mail vous a été envoyé à l’adresse suivante, sauf si un compte ${app_name} lui a déjà été associé :</string>
-    <string name="tchap_register_wait_for_email_subtitle">Merci de cliquer sur le lien proposé dans cet e\u2011mail afin de terminer la création de votre compte. Vous pourrez alors vous connecter en allant sur l’écran de connexion.</string>
-    <string name="tchap_register_wait_for_email_login_button">Aller à l’écran de connexion</string>
-    <string name="tchap_register_wait_for_email_no_email">Je n’ai pas reçu l’email!</string>
+    <string name="tchap_register_wait_for_email_subtitle">Merci de cliquer sur le lien proposé dans cet e\u2011mail afin de terminer la création de votre compte.</string>
     <string name="tchap_password_weak_pwd_error">Ce mot de passe est trop faible. Il doit contenir au moins 8 caractères, avec au moins un caractère de chaque type : majuscule, minuscule, chiffre, caractère spécial.</string>
-    <string name="tchap_login_reset_password_warning_content">Le changement de mot de passe réinitialisera toutes les clés de chiffrement sur toutes vos sessions, rendant l’historique des discussions chiffrées illisible. Exportez vos clés de salon depuis une autre session avant de réinitialiser votre mot de passe.</string>
```

Comment:
Please check this text in element strings. we should probably use this one instead.